website/docs/widgets/padding.md

```diff
@@ -0,0 +1,30 @@
+# Padding
+
+Mirai padding allows you to build the Flutter padding widget using JSON. 
+To know more about the padding widget in Flutter, refer to the [official documentation](https://api.flutter.dev/flutter/widgets/Padding-class.html).
+
+## Properties
+
+| Property | Type              | Description                                       |
+| --- |-------------------|---------------------------------------------------|
+| padding | `MiraiEdgeInsets` | The amount of space by which to inset the child. |
+| child | `Map<String,dynamic>` | The widget below this widget in the tree. |
+
```

Comment:
Hey @khanjasir90, In note or description can we also mention other ways to give padding?
1. All padding: {"padding": 12}
2. Padding fromLTRB: {"padding": [8, 12, 8, 12]}.